index.js

```diff
@@ -29,10 +68,35 @@ server.post("/api/login", (req, res) => {
     .first()
     .then(user => {
       if (user && bcrypt.compareSync(creds.password, user.password)) {
-        res.status(200).send("Logged in");
+        req.session.username = user.username;
+        res.status(200).send(`Welcome, ${req.session.username}`);
       } else {
         res.status(401).json({ message: "Username or password is incorrect" });
       }
     });
 });
+
+server.get('/api/logout', (req, res) => {
+  if (req.session) {
+    req.session.destroy(err => {
+      if (err) {
+        res.send('error logging out');
+      } else {
+        res.send('good bye');
+      }
+    });
+  }
+});
+
+//Admins
+
+server.get("/api/admins", (req, res) => {
+  db("admins")
+    .select()
+    .then(admins => {
+      res.status(200).json(admins);
+    })
+    .catch(err => res.status(500).json(err));
+});
```

Comment:
This is not part of MVP, but for funsies, how would you implement role authorization with cookies/sessions?